engine_modules/corporation_run/tests/test_signals.py

```diff
@@ -26,6 +27,22 @@ def setUp(self):
 		)
 		self.eo.save()
 
+		self.so = SabotageOrder(
```

Comment:
don't define it on self, you're only using them once

(code in setUp is ran once for each test)